src/test/java/com/github/iotexproject/antenna/protocol/IOTXTest.java

```diff
@@ -30,4 +30,27 @@ public void testTransfer() {
         String hash = iotx.sendTransfer(request);
         Assert.assertNotNull(hash);
     }
+
+    @Test
+    public void testStakeCreate() {
+        Account account = IotexAccount.create("cfa6ef757dee2e50351620dca002d32b9c090cfda55fb81f37f1d26b273743f1");
+        IOTX iotx = new IOTX(RPCMethodTest.IOTEX_CORE);
+
+        StakeCreateRequest request = new StakeCreateRequest();
+        request.setNonce(0l); // optional, can be null
+        request.setGasLimit(1000000l); // optional, can be null
+        request.setGasPrice("10"); // optional, can be null
+        request.setAccount(account);
+        request.setCandName("io19d0p3ah4g8ww9d7kcxfq87yxe7fnr8rpth5shj");
+        request.setAmount("100");
+        request.setDuration(10000);
+        request.setAutoStake(true);
+        request.setPayload("7061796c6f6164"); // optional, can be null
+
+        String hash = iotx.stakeCreate(request);
+        Assert.assertEquals(
+                "f1785e47b4200c752bb6518bd18097a41e075438b8c18c9cb00e1ae2f38ce767",
+                hash)
```

Comment:
this test value is from our back-end for trust-wallet

https://github.com/trustwallet/wallet-core/commit/0d91994784f386f7373c66427131e107025ff791#diff-17b6a4326ae3240c87dffebd4a11a111R164